backend/static/js/qleverUI.js

```diff
@@ -6,11 +6,12 @@ var predicateNames = {};
 var objectNames = {};
 var high_query_time_ms = 100;
 var very_high_query_time_ms = 1000;
-var runtime_log = [];
-var query_log = [];
-var lastQueryUpdate = { queryId: null, updateTimeStamp: 0 };
+// Map guarantees to keep insertion order
```

Comment:
// Note: when we later iterate over this `Map`, we get the key-value pairs in the order in which the keys were first inserted.

Reply:
Better move this comment to line 808 (or so), where we actually iterate over the map.